### DISCUSSION TITLE
Welcome to highly-skilled-job-offers Discussions!

### DISCUSSION THREAD

**Discussion by coditori** [coditori/highly-skilled-job-offers#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MohammadRezaei92** [comment]
سلام
من یک پیشنهاد اصلاحیه در مورد آیتم زیر دارم
Change your Linkedin location to the country you wanna move
من این کار رو انجام دادم و تجربه ای که داشتم اینطور بود:
درسته با وانمود کردن به بودن در کشور مقصد پیشنهاد های بیشتری از طرف رکروتر ها میگیرید ولی در نهایت مجبور میشید حقیقت رو بهشون بگین و وقتی متوجه میشن شما محل زندگیتون رو دروغ گفتین حس میکنن میخواستین فریبشون بدین و در نهایت این کار به ضرر شما تموم میشه. مثل اتفاقی که برای خودم افتاد و هنگام مصاحبه با ریکروتر آلمانی به محض اینکه فهمید محل زندگیم اروپا نیست دیگه بحث رو ادامه نداد و فقط شروع به نصیحت در مورد دادن اطلاعات درست کرد. کلا ناجور میشه قضیه :)
پیشنهاد میکنم این بند حذف بشه. ممنون

**Reply by coditori** [reply]
Thanks for your comment, and sorry for the late reply. This approach may not work for Iranians, but it can still be helpful for others. I'll mention your concern in the repo.

**Comment by arashabbac** [comment]
سلام

من همون روز به ایران تغییرش دادم به همین دلیلی که فرمودین
ممنون بابت اطلاع رسانیتون
